testhelper/docker/resource/transformer/transformer_test.go

```diff
@@ -0,0 +1,57 @@
+package transformer_test
+
+import (
+	"bytes"
+	"io"
+	"net/http"
+	"net/url"
+	"testing"
+
+	"github.com/ory/dockertest/v3"
+	"github.com/stretchr/testify/require"
+
+	"github.com/rudderlabs/rudder-go-kit/testhelper/docker/resource/transformer"
+)
+
+func TestSetup(t *testing.T) {
+	pool, err := dockertest.NewPool("")
+	require.NoError(t, err)
+
+	transformerContainer, err := transformer.Setup(pool, t, transformer.WithConfigBackendURL("https://api.dev.rudderlabs.com"))
```

Comment:
Is it too complex to use a mocked HTTP server here? It would be nice to have these integration tests not depend on an actual call to control plane.

Reply:
Not sure if it is right thing to do, as mock http server would have to know internal working of transformer. If internal working of transformer changes mock http server would also needs to be changed.

Reply:
Not a blocking comment, I was asking merely if it's something easy to do because it makes just a simple HTTP request then we can mock it and avoid the dependency. If it starts making too many requests then ofc it's not worth it. You could use a simple [TCP Proxy](https://github.com/rudderlabs/rudder-go-kit/tree/main/tcpproxy) to find out as a little experiment :slightly_smiling_face: